apps/aehttp/priv/oas3.yaml

```diff
@@ -3378,6 +3378,8 @@ components:
       properties:
         reason:
           type: string
+        error_code:
+          type: string
       required:
         - reason
```

Comment:
`error_code` was added in other places, but there are still cases when it is missed? so it can't be required?

Reply:
It was only added where there is anything interesting to report I think.

Reply:
And it is not required. You can get responses without any `error_code`. It's just a bit of extra information the SDK can look at when deciding what to do.